docs/modules/did_session.md

```diff
@@ -23,7 +23,7 @@ import { DIDSession } from 'did-session'
 import { EthereumWebAuth, getAccountId } from '@didtools/pkh-ethereum'
 
 const ethProvider = // import/get your web3 eth provider
-const addresses = await ethProvider.enable()
+const addresses = await (ethProvider as any).request({ method: 'eth_requestAccounts' })
```

Comment:
Don't think the example is using typescript?

Reply:
Right, sorry.

Reply:
all the snippets are marked as ts in markdown, but wasnt very intentionally chosen, mostly just because types were sometimes referenced but mostly the snippets are just plain js

Reply:
I can convert all the snippets to `js` then :). Did I understand correctly that this is what we want?

Reply:
yeah think that makes sense here